data/entities.yaml

```
@@ -816,6 +816,22 @@
   properties: [portable]
   capabilities: [log]
 
+- name: hearing aid
+  display:
+    attr: device
+    char: '@'
+  description:
+  - "Allows robot to hear what nearby robots are saying."
+  - "Simply having this device installed will automatically
+     add messages said by nearby robots to this robots log,
+     assuming it has a logger installed."
+  - "That way you can view any heard message later either in
+     the logger or the message window."
+  - "To wait for a message and get the string value, use:"
+  - "`l <- listen; log $ \"I have waited for someone to say \" ++ l`"
+  properties: [portable]
+  capabilities: [log]
```

Should this be `listen`?

---

Right... :sweat_smile: 
```suggestion
  capabilities: [listen]
```